nimble/host/src/ble_l2cap_coc.c

```diff
@@ -499,6 +499,34 @@ ble_l2cap_coc_send(struct ble_l2cap_chan *chan, struct os_mbuf *sdu_tx)
     return ble_l2cap_coc_continue_tx(chan);
 }
 
+uint16_t
+ble_l2cap_get_scid(struct ble_l2cap_chan *chan)
```

Comment:
I suggest to change return type from uint16_t to int and in case of chan==NULL return -1.
This is now API for the application so let's be on the safe side.